tools/test/examples/examples.json

```diff
@@ -21,34 +19,33 @@
     {
       "name": "mbed-os-example-blinky-baremetal",
       "github": "https://github.com/ARMmbed/mbed-os-example-blinky-baremetal",
-      "mbed": [
-          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-blinky-baremetal"
-      ],
-      "test-repo-source": "github",
+      "sub-repo-example": false,
+      "subs": [],
       "features" : [],
       "targets" : ["K66F", "NUCLEO_F429ZI", "ARCH_PRO", "LPC1768"],
       "toolchains" : [],
       "exporters": ["iar", "make_armc5", "make_armc6", "make_gcc_arm", "make_iar"],
       "compile" : true,
       "export": true,
+      "test" : false,
       "auto-update" : true
     },
     {
       "name": "mbed-os-example-tls",
       "github": "https://github.com/ARMmbed/mbed-os-example-tls",
-      "mbed": [
-          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-tls-benchmark",
-          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-tls-tls-client",
-          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-tls-hashing",
-          "https://os.mbed.com/teams/mbed-os-examples/code/mbed-os-example-tls-authcrypt"
+      "sub-repo-example": true,
```

Comment:
Nice to have finally support for these. Can this be automated? If this is true, we walk through the root and if mbed-os.json file found, it is an example for testing ? This way we do not need to define subs (this can be out of date easily as everytime someone adds a new  sub example or restructure example, this breaks). 

Or any other way to do this without having this "subs" in here?

Reply:
Removing these lines breaks the release script

Reply:
> 
> 
> Nice to have finally support for these. Can this be automated? If this is true, we walk through the root and if mbed-os.json file found, it is an example for testing ? This way we do not need to define subs (this can be out of date easily as everytime someone adds a new sub example or restructure example, this breaks).
> 
> Or any other way to do this without having this "subs" in here?
@0xc0170 That is how the release script currently updates the mbed-os.lib files. It works down the route. 

@jamesbeyond Actually looking at the release script more carefully I think this change is ok. The script only looks at the 'name' field so as long as that doesn't change we should be ok!